### PR TITLE
Upgrade go modules in go.mod

### PR DESCRIPTION
**Pull request by juniortaeza**
### Desired Outcome

This pull request upgrades go.mod and go.sum

### Implemented Changes

- upgrades golang.org/x/text to v0.3.8
- upgrades golang.org/x/sys to v0.8.0
- golang.org/x/crypto is already at proper version

#### Changelog

- [X] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a
  CHANGELOG update

#### Test coverage

- [ ] This PR includes new unit and integration tests to go with the code
  changes, or
- [X] The changes in this PR do not require tests

#### Documentation

- [ ] Docs (e.g. `README`s) were updated in this PR
- [ ] A follow-up issue to update official docs has been filed here: [insert issue ID]
- [X] This PR does not require updating any documentation

#### Behavior

- [ ] This PR changes product behavior and has been reviewed by a PO, or
- [X] These changes are part of a larger initiative that will be reviewed later, or
- [ ] No behavior was changed with this PR

#### Security

- [ ] Security architect has reviewed the changes in this PR,
- [X] These changes are part of a larger initiative with a separate security review, or
- [ ] There are no security aspects to these changes

### CODE REVIEW

**Review by gl-johnson — COMMENTED** [review]
The go.sum is showing this version for golang.org/x/crypto: `v0.0.0-20210921155107-089bfa567519` which seems to be a lower version than we want. You may need to add an  additional replace statement and `go mod tidy` to fix it

Additionally - any lower versions of `golang.org/x/text` and `golang.org/x/sys` that are still showing up in go.sum after the changes you've made may also require explicit replaces to the new version

**Comment by juniortaeza** [comment]
Updated! Thanks @gl-johnson

File: go.mod [hunk 1]
```diff
@@ -168,19 +170,47 @@ replace golang.org/x/net v0.0.0-20211112202133-69e39bad7dc2 => golang.org/x/net
 
 replace golang.org/x/net v0.0.0-20220722155237-a158d28d115b => golang.org/x/net v0.7.0
 
-replace golang.org/x/text v0.0.0-20170915032832-14c0d48ead0c => golang.org/x/text v0.3.7
+replace golang.org/x/text v0.0.0-20170915032832-14c0d48ead0c => golang.org/x/text v0.3.8
```
Comment (gl-johnson) [inline go.mod:173]: Just curious if there is something preventing us from replacing these with golang.org/x/text v0.9.0 which is the version we specify in our dependencies now.

Comment (juniortaeza) [reply go.mod:173]: I did notice v0.9.0 to be the most recent one but I went with the minimum requirement listed in the ticket. I think we should keep things in harmony so I'll change these to v0.9.0 as well!

**Review by gl-johnson — APPROVED** [review]

**Pull request merged**